client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -168,7 +168,9 @@ export default function CampaignItemDetails( props: Props ) {
 			value: views_organic || 0,
 		},
 	];
-	const databarTotal = databars.reduce( ( total, item ) => total + item.value, 0 );
+	const databarTotal = databars
+		.map( ( item ) => ( item.value > 0 ? item.value : 0 ) ) // Get only positive values.
```

Comment:
It's easier to have this code in one iteration. For example, you could move the check in the reduce like this:
```
.reduce( ( total, value ) => value > 0 ?  total + value : total, 0 );
```
Its the same as before, but without that extra iteration that the map function adds. 

Nonetheless, this is a nitpick of mine, there isn't any performance consideration in a two elements array. Fell free to ignore this.

Reply:
I think that @jjolmo has made some changes in the backend to not result in negative numbers.. but having a check in the front is also useful

Reply:
Initially I thought that ternary operator will make it look more complex, but it look good. Thanks for the suggestion, @sbarbosa. Applied in  https://github.com/Automattic/wp-calypso/pull/78476/commits/39a2bbdc5ab260c28b4eec3cfd3bcead49774e24.